src/neo/SmartContract/InteropService.Contract.cs

```diff
@@ -174,6 +176,10 @@ private static bool Contract_CallEx(ApplicationEngine engine, UInt160 contractHa
                         context_new.EvaluationStack.Push(args[i]);
                     context_new.InstructionPointer = md.Offset;
                 }
+
+                md = contract.Manifest.Abi.GetMethod("_initialize");
+                if (md != null) engine.LoadClonedContext(md.Offset);
```

Comment:
Should the eval stack of `_initialize` context be empty?

Reply:
Why?

Reply:
Both can influence each other, as we have already put the arguments on the eval stack.

Reply:
If one contract A invokes another contract B many times, every time we'll load the `_initialize` of B, it's not very efficient. 

How about creating a `.sdata`(static data) segment in `NEF` file, where to put the static variables, when we load a contract, we can just pass the data into static slot.

Reply:
They are static variables, not constants.

Reply:
> They are static variables, not constants.

It doesn't matter, the static variables will not be persisted, which will not affect the next time to load.

Reply:
I'm wondering whether the contract on the chain is stateful or stateless. If it is stateless, then we can call `_initialize` every time. If it is stateful, `_initialize` should only be called once.

Reply:
```js
contract A{
  int a = 0;

  function inc(){
      return ++a
  }
}
```

If a contract is stateful, when call `inc` method, it'll increase the `a`. 
If a contract is stateless, when call `inc` method, it'll only return 1 every time.  

This will affect how the compiler treats the contract variables.